README.markdown

```diff
@@ -5621,14 +5621,17 @@ ngx.sleep
 
 **syntax:** *ngx.sleep(seconds)*
 
-**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;*
+**context:** *init_by_lua&#42;, init_worker_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;, exit_worker_by_lua&#42;*
```

Comment:
It's not safe to use blocking sleeps in certain contexts like `init_worker_by_lua*`, `set_by_lua*`, `header_filter_by_lua*`, and etc.

I'm fine with enabling it in contexts where blocking operations do not matter, like in `init_by_lua*` and `exit_worker_by_lua*`. Otherwise it's too dangerous.